packages/app/src/app/pages/common/Modals/NewTeamModal/TeamSubscription.tsx

```diff
@@ -56,6 +57,13 @@ export const TeamSubscription: React.FC = () => {
     pageMounted();
   }, [pageMounted]);
 
+  React.useEffect(() => {
```

Comment:
Might be an unnecessary abstraction, but we could have a custom hook that accepts just the event name and calls the track function with the params on a useEffect when the component mounts